docs/scenarios_authoring/Enrichers.md

```diff
@@ -41,6 +41,28 @@ DatabaseLookupEnricher is a specialized look-up component; it returns all column
 
 ![alt_text](img/databaseLookupEnricher.png "databaseLookup Enricher")
 
+### DecisionTable enricher
+
+This component allows the user to match input to a pre-defined result. Please be wary that this component works a bit different than a classical decision table([Decision tables in Wikipedia](https://en.wikipedia.org/wiki/Decision_table)). It consists of the fields:
+- Name: the name of the component.
```

Comment:
is it a component name or rather a node name?

Reply:
Name of the specific component, which is also a node in the graph.

Reply:
The documentation you refer to "A node is a component used in a scenario". I can change it to "name of the particular node" if you want to be specific about the fact that this is the name of node instance.

Reply:
Name and description are parameters common for every component. WDYT, we should describe these parameters for every component type or we should describe it in one place?